GA/objectives.py

```
@@ -0,0 +1,361 @@
+            This is computed as the product of the power and the time spent in the flight phase.
+        """
+
+        if isinstance(outputs, list):
+            # If outputs is a list, we are solving for a multi-point optimisation problem, so we sum the total energ for all points.
+            energy = 0
+            for idx, output in enumerate(outputs):
+                power = output["data"]["Total power CP"] * (0.5 * self.oper[idx]["atmos"].density[0] * self.oper[idx]["Vinl"] ** 3 * self.Lref ** 2)
+                energy += power * self.oper[idx]["flight_phase_time"]
+        else:
+            # If outputs is a single dictionary, we are solving for a single-point optimisation problem.
+            power =  outputs["data"]["Total power CP"] * (0.5 * self.oper["atmos"].density[0] * self.oper["Vinl"] ** 3 * self.Lref ** 2)
```

Flake8 (7.2.0)

[error] multiple spaces after operator

(E222)